vaadin-text-field-flow-integration-tests/src/main/java/com/vaadin/flow/component/textfield/tests/ValueChangeModePage.java

```diff
@@ -47,17 +53,14 @@ private void initView() {
         message.setId("message");
         add(message);
 
-        TextField textField = new TextField();
-        addField(textField);
-        addButtons(textField);
-
-        TextArea textArea = new TextArea();
-        addField(textArea);
-        addButtons(textArea);
+        Stream.of(new TextField(), new TextArea(), new PasswordField(),
+                new NumberField(), new EmailField(), new IntegerField(),
+                new BigDecimalField()).forEach(this::setupTestComponent);
+    }
```

Comment:
Refactored this a bit and added the value-change-mode tests also for other fields, which didn't have it yet.